services/src/test-library/src/main/java/org/openkilda/testing/config/DefaultServiceConfig.java

```diff
@@ -80,6 +80,11 @@ public RestTemplate mininetFlowToolRestTemplate(@Value("${mininet-flowtool.endpo
         return buildLoggingRestTemplate(endpoint);
     }
 
+    @Bean(name = "otsdbRestTemplate")
+    public RestTemplate otsdbRestTemplate(@Value("${otsdb.host}") String host, @Value("${otsdb.port}") int port) {
+        return buildLoggingRestTemplate(String.format("http://%s:%s", host, port));
```

Comment:
I'd rather not hardcode protocol - it could easily move to https

Reply:
fixed in https://github.com/telstra/open-kilda/pull/1422/commits/5a55eb977f344b80d4b2becbb83f6b2b26ae095d